plugins/broadcom_symantec_endpoint_protection/icon_broadcom_symantec_endpoint_protection/util/api.py

```diff
@@ -110,7 +113,7 @@ def new_client(
 
         auth_body = {"username": username, "password": password, "domain": domain}
         headers = {"Content-Type": "application/json"}
-        response = requests.post(url=auth_url, json=auth_body, headers=headers, verify=False)  # noqa: B501
+        response = requests.post(url=auth_url, json=auth_body, headers=headers, verify=ssl_verify)  # noqa: B501
```

Comment:
Can this be removed now that we don't manually set it to False
```suggestion
        response = requests.post(url=auth_url, json=auth_body, headers=headers, verify=ssl_verify) 
```